code/modules/clothing/accessories/holomap.dm

```diff
@@ -34,6 +34,25 @@ var/list/holomap_cache = list()
 	holomap_color = "#13B40B"
 
 
+/obj/item/clothing/accessory/holomap_chip/ert
+	name = "emergency response team holomap chip"
+	icon_state = "holochip_ert"
+	marker_prefix = "ert"
+	holomap_filter = HOLOMAP_FILTER_ERT
+	holomap_color = "#5FFF28"
+
+	var/list/prefix_update = list(
```

Comment:
Byond allows from string-indexed lists, which is what you should really do for this i.e.

`list("/obj/item/whatever" = "ertc", ...`